src/database/models/__init__.py

```diff
@@ -37,6 +37,15 @@ def __str__(self):
         return self.name
 
 
```

Comment:
Please include the migration python file as well.

Reply:
Thanks for noticing the missing migration script! Added it